src/ChatList/ChatLoader/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useState, useRef, useEffect } from 'react';
+
+import { Props } from './props';
+import { styles } from './styles';
+
+export const ChatLoader: React.FC<Props> = ({
```

Comment:
what is React.FC?

Reply:
React's Functional Component interface 👍